subt_ign/src/ConnectionHelper.cc

```diff
@@ -97,7 +97,34 @@ std::map<std::string, std::vector<ignition::math::Vector3d>>
     {"Cave Corner 30 D Type B", {{12.5, 0, 0}, {0, 12.5, 0}}},
     {"Cave Corner 30 D Lights Type B", {{12.5, 0, 0}, {0, 12.5, 0}}},
     {"Cave Corner 30F D Type B", {{12.5, 0, 0}, {0, -12.5, 0}}},
-    {"Cave Corner 30F D Lights Type B", {{12.5, 0, 0}, {0, -12.5, 0}}}
+    {"Cave Corner 30F D Lights Type B", {{12.5, 0, 0}, {0, -12.5, 0}}},
+
+    {"Cave 2 Way 01 Type A", {{-25.0, 0.0, 0.0}, {25.0, 0.0, 0.0}}},
+    {"Cave 3 Way 01 Type A", {{0.0, -25.0, 0.0}, {-25.0, 50.0, 0.0}, {50.0, 25.0, 0.0}}},
+    {"Cave 3 Way 02 Type A", {{-50.0, 25.0, 0.0}, {-25.0, 50.0, 0.0}, {0.0, -25.0, 0.0}}},
+    {"Cave 3 Way Elevation 01 Type A", {{0.0, 50.0, 0.0}, {-100.0, -50.0, -25.0}, {0.0, -50.0, -25.0}}},
+    {"Cave 3 Way Elevation 02 Type A", {{0.0, -50.0, 0.0}, {-100.0, -50.0, -25.0}, {0.0, 50.0, 0.0}}},
+    {"Cave 3 Way Elevation 03 Type A", {{0.0, 50.0, 75.0}, {50.0, 0.0, -50.0}, {0.0, -50.0, 0.0}}},
+    {"Cave 4 Way 01 Type A", {{0.0, 25.0, 0.0}, {25.0, 0.0, 0.0}, {0.0, -25.0, 0.0}, {-25.0, 0.0, 0.0}}},
+    {"Cave Corner 01 Type A", {{25.0, 0.0, 0.0}, {0.0, -25.0, 0.0}}},
+    {"Cave Corner 02 Type A", {{0.0, 25.0, 0.0}, {25.0, -50.0, 0.0}}},
+    {"Cave Corner 03 Type A", {{-25.0, 0.0, 0.0}, {0.0, -25.0, 0.0}}},
+    {"Cave Corner 04 Type A", {{0.0, 50.0, 25.0}, {-50.0, -50.0, 0.0}}},
+    {"Cave Cap Type A", {{0.0, 0.0, 0.0}}},
+    {"Cave Elevation 01 Type A", {{0.0, -50.0, 0.0}, {0.0, 50.0, 75.0}}},
+    {"Cave Elevation 02 Type A", {{-50.0, 0.0, 75.0}, {50.0, 0.0, 0.0}}},
+    {"Cave Elevation Corner Type A", {{25.0, 75.0, 25.0}, {0.0, -100.0, 0.0}}},
+    {"Cave Elevation Straight Type A", {{0.0, 50.0, 0.0}, {0.0, -50.0, 0.0}}},
+    {"Cave Split Type A", {{0.0, -50.0, 0.0}, {0.0, 50.0, 0.0}}},
+    {"Cave Straight Shift Type A", {{-25.0, 25.0, 0.0}, {25.0, -25.0, 0.0}}},
+    {"Cave Straight Type A", {{0.0, 25.0, 0.0}, {0.0, -25.0, 0.0}}},
+    {"Cave U Turn 01 Type A", {{-25.0, -25.0, 0.0}, {25.0, -25.0, 0.0}}},
+    {"Cave U Turn Elevation Type A", {{-50.0, -25.0, 0.0}, {0.0, -25.0, 25.0}}},
+    {"Cave Vertical Shaft Cantilevered Type A", {{0.0, -5.46392e-07, 10.0}, {0.0, 50.0, 25.0}}},
+    {"Cave Vertical Shaft Straight Bottom Type A", {{0.0, 0.0, 10.0}, {0.0, 25.0, 0.0}, {0.0, -25.0, 0.0}}},
+    {"Cave Vertical Shaft Straight Top Type A", {{0.0, 25.0, 0.0}, {0.0, -25.0, 0.0}, {0.0, 0.0, 0.0}}},
+    {"Cave Vertical Shaft Type A", {{0.0, 0.0, 10.0}, {0.0, 0.0, 25.0}}},
+    {"Cave Vertical Shaft Dead End Type A", {{0.0, 0.0, 10.0}}}
```

Comment:
add transition tile

Reply:
> add type A tiles with lights?

Added in 8a31402ba00787fca6183038423a5c71c88c594b